benchmarks/run.py

```diff
@@ -198,6 +188,26 @@ def main():
         csv_results.append([p, test_class.version, avg])
         print()
 
+    return results, csv_results
+
+def main():
+    json_path = THIS_DIR / 'cases.json'
+    if not json_path.exists():
+        print('generating test cases...')
+        cases = [generate_case() for _ in range(2000)]
+        with json_path.open('w') as f:
+            json.dump(cases, f, indent=2, sort_keys=True)
+    else:
+        with json_path.open() as f:
+            cases = json.load(f)
+
+    tests = [TestPydantic]
+    if 'pydantic-only' not in sys.argv:
+        tests += other_tests
+
+    repeats = int(os.getenv('BENCHMARK_REPEATS', '5'))
+    results, csv_results = run_tests(tests, cases, repeats, 'json' in sys.argv)
```

Comment:
```suggestion
    test_json = TEST_JSON in os.environ
    results, csv_results = run_tests(tests, cases, repeats, test_json)
```

That way:
1. it's consistent with `BENCHMARK_REPEATS`
2. you can alter it from outside the makefile.